R/build-reference.R

```diff
@@ -36,6 +36,12 @@
 #' You can also select topics that contain specified Rd concepts with
 #' `has_concept("blah")`.
 #'
+#' You can also select topics that do not contain any of a vector of specified
```

Comment:
Can you please merge with the previous paragraph?

Reply:
Yes.